tests/lib/Service/OpenProjectAPIServiceTest.php

```diff
@@ -629,6 +629,43 @@ private function getStorageMock($nodeClassName = null) {
 		return $storageMock;
 	}
 
+	/**
+	 * @param array<string, object> $constructParams
```

Comment:
maybe over explaining, bu this is how I understood what the function does

```suggestion
     * generates a list mocks that can be given as arguments to the constructor of OpenProjectAPIService
     * by default only empty mocks are generated, but specific mocks can be passed in using the 
     * $constructParams parameter
     * 
	 * @param array<string, object> $constructParams specific mocks for the constructor of OpenProjectAPIService. Format has to be [[<string>, <string>]] with the first being the constructor parameter name and the second one the mock
```

Reply:
I have added the docs but they are a little bit different. Please have a look.